### ISSUE TITLE
AWAKE Protocol: PIN validation step changes to allow choosing a device to link with

### ISSUE THREAD

**Issue by matheus23** [fission-suite/webnative#391]

**Comment by bgins** [comment]
> This does not describe spec changes. (Maybe someone misread "PIN validation step changes" for "spec changes"?)  

Agreed, the whitepaper looks like it holds for these changes. Multiple devices does seem like an implementation detail. Same with the method by which the user confirms the pin (displayed to the user vs user enters pin).
